### DISCUSSION TITLE
Grove - CO2 & Temperature & Humidity Sensor (SCD41) | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1100] [Q&A]
# Grove - CO2 & Temperature & Humidity Sensor (SCD41) | Seeed Studio Wiki

Grove - CO2 & Temperature & Humidity Sensor (SCD41)

https://wiki.seeedstudio.com/Grove-CO2_&_Temperature_&_Humidity_Sensor-SCD41/

**Comment by kornpow** [comment]
So far I have been unable to get multiple SCD41 devices to appear on my ESP32S3 I2C bus. I know it isn't the XIAO because I can put another sensor (of a different type) on it, and that sensor does show up on the bus.

I've tried various different I2C bus speeds. Multiple grove i2c cables. Multiple sensor modules. Please help!
